README.md

```diff
@@ -34,13 +34,26 @@ sim.add_partner(
   birthday: Date.new(1966, 5, 5)
 )
 
+sim.set_growth_rates(
```

Comment:
Ok here's a thought, and sorry I'm bouncing around a lot.  But given that this is the primarily variable for each monte carlo, should we passing these growth rates into the `.simulate` method instead of setting them separately?

Reply:
Yeah, I was thinking about that. Most of the simulations' parameters are set up to be fixed, but growth rates (and possibly some other variables, down the line?) are meant to be variable. The API above makes sense for a simulator meant to be run exactly once, but once we move towards focusing on Monte Carlo, it will become kinda clunky. What do you think about either of these options?

```
sim.simulate do |run|
  run.set_growth_rates(...)
end
```

```
sim.simulate(growth_rates: ...)
```

Reply:
Great.  I think the latter is fine.  I don't expect to be passing in any other settings at this point since they're all above.